lib/core/constants/markers.dart

```diff
@@ -2,19 +2,19 @@ import 'package:google_maps_flutter/google_maps_flutter.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 List<Marker> markers(AppIntl intl) => [
-  Marker(
-    position: const LatLng(45.49511855948888, -73.56270170940309),
-    markerId: MarkerId('security_building_A'),
-    infoWindow: InfoWindow(title: intl.security_station),
-  ),
-  Marker(
-    position: const LatLng(45.495089693692194, -73.56374294991838),
-    markerId: MarkerId('security_building_B'),
-    infoWindow: InfoWindow(title: intl.security_station),
-  ),
-  Marker(
-    position: const LatLng(45.49391646843658, -73.5634878349083),
-    markerId: MarkerId('security_building_D'),
-    infoWindow: InfoWindow(title: intl.security_station),
-  ),
-];
+    Marker(
+      position: const LatLng(45.49511855948888, -73.56270170940309),
+      markerId: const MarkerId('security_building_A'),
+      infoWindow: InfoWindow(title: intl.security_station),
+    ),
+    Marker(
+      position: const LatLng(45.495089693692194, -73.56374294991838),
+      markerId: const MarkerId('security_building_B'),
+      infoWindow: InfoWindow(title: intl.security_station),
+    ),
+    Marker(
+      position: const LatLng(45.49391646843658, -73.5634878349083),
+      markerId: const MarkerId('security_building_D'),
+      infoWindow: InfoWindow(title: intl.security_station),
+    ),
+  ];
```

Comment:
If there is no changes made in this file remove it from the PR

Reply:
I added the `const` keyword over the MarkerId, it seems that before it wasn't require but now for some reason I got some warning about it. Don't know if on intellij it's doing the same thing though. What do you think?
![image](https://user-images.githubusercontent.com/25663435/116801065-689f6f80-aad4-11eb-9d3c-de7d9c154924.png)

Reply:
Ohh great, I didn't pay attention to the word const. Keep it for sure!!